handlers/adjtime.yml

```diff
@@ -1,3 +1,4 @@
 ---
 - name: Write adjtime
   command: hwclock --systohc
+  when: server_type == "hw"
```

Comment:
we should document things like `server_type`, especially if it is set in another role or the inventory.